core/asset/service.go

```diff
@@ -164,15 +169,22 @@ func isValidUUID(u string) bool {
 	return err == nil
 }
 
-func buildNodeAttrs(assetProbes map[string][]Probe) map[string]NodeAttributes {
+func buildNodeAttrs(assetProbes map[string][]Probe, attributes map[string]Asset) map[string]NodeAttributes {
 	nodeAttrs := make(map[string]NodeAttributes, len(assetProbes))
 	for urn, probes := range assetProbes {
-		if len(probes) == 0 {
-			continue
+		var probesInfo ProbesInfo
+		if len(probes) > 0 {
+			probesInfo = ProbesInfo{Latest: probes[0]}
+		}
+
+		attrs := make(map[string]interface{})
```

Comment:
This would mean that attributes are included only if probes are present for the asset. Would recommend adding test cases to catch such issues.

Reply:
Yes. Test cases are pending. will add this test case.